schema_based_on_diagram.sql

```diff
@@ -0,0 +1,69 @@
+-- creating patients table
+CREATE TABLE patients (
+    id INT PRIMARY KEY,
+    name VARCHAR(255) NOT NULL,
+    date_of_birth DATE
+);
+
+-- Create the medical_histories table
+CREATE TABLE medical_histories (
+    id INT PRIMARY KEY,
+    admitted_at TIMESTAMP,
+    patient_id INT,
+    status VARCHAR(255)
+);
+
+-- Add a foreign key constraint to the medical_histories table
+ALTER TABLE medical_histories
+ADD CONSTRAINT fk_patient
+FOREIGN KEY (patient_id)
+REFERENCES patients(id);
+
+-- Create the treatment table
+CREATE TABLE treatment (
+    id INT PRIMARY KEY,
+    type VARCHAR(55),
+    name VARCHAR(255) NOT NULL,    
+);
+
+--Add a foreign key constraint to treament table
+ALTER TABLE treatment 
+ADD CONSTRAINT fk_medical_histories
+FOREIGN KEY (id)
+REFERENCES medical_histories(id);
```

Comment:
- [ ] Its best practice and advisable to create a separate table to show the "join table" for the many-to-many relationships, even if they are not shown on the diagram. 
- [ ] Kindly create a table between `medical_histories` and `treatments`  

### Sample implementation
```SQL
  CREATE TABLE medical_histories_has_treatments (
    medical_history_id int refrences medical_histories(id),
    treatment_id int refrences treatments(id),
    );

```